### DISCUSSION TITLE
mod合集包 快速配置版本

### DISCUSSION THREAD

**Discussion by CTDE42** [Meloong-Git/PCL#4563] [提问] [answered]
希望能增加一个能添加mod包并定制mod进去，下载任意版本的时候检查选中mod包里的mod有没有要下载Minecraft版本的对应版本mod后安装进去，快速配置，或者说直接加一个快速配置功能 版本设置一条龙

**Comment by CTDE42** [comment]
比如优化全家桶，方便快速配置

**Comment by Pigeon0v0** [comment]
#1646

**Comment by WForst-Breeze** [comment, marked as answer]
Author 指的应该是 #728？

但 #1646 的确能满足一部分需求，就是不能做到一键即完成就是了（
